project/src/test/java/com/google/sps/data/PlacesFetcherTest.java

```diff
@@ -40,34 +40,49 @@
 @RunWith(JUnit4.class)
 public final class PlacesFetcherTest {
 
-  private static final String PLACE_URL = "https://www.google.com/"; // used for Places
-  private static final URL PLACES_DETAILS_URL = createTestURL(PLACE_URL); // used for PlaceDetails
+  private static final String PLACE_WEBSITE = "https://www.google.com/"; // used for Places
+  private static final String PLACE_GOOGLE_URL = "https://www.google.com/"; // used for Places
```

Comment:
Make this a different string. This way, if you ended up mixing the two strings, the test will find that out.

Reply:
Right.